apps/dotcom/src/utils/sharing.ts

```diff
@@ -96,7 +96,8 @@ export async function getNewRoomResponse(snapshot: Snapshot) {
 
 export function useSharing(): TLUiOverrides {
 	const navigate = useNavigate()
-	const id = useSearchParams()[0].get('id') ?? undefined
+	const params = useParams()
+	const roomId = params.roomId
```

Comment:
Looks like we lost the `parentSlug` logic for the snapshots when migrating to react router. We were looking at search params instead of regular params.